docs/user-settings.md

```diff
@@ -21,12 +22,21 @@ meta -->
 
 This document explains how to configure a User Settings dialog in the top navigation of Luigi.
 
-- [Overview](#overview)
-- [Example](#example)
-- [Parameters](#parameters)
-- [Write a custom editor](#write-a-custom-editor)
-- [Customize the user settings dialog](#customize-the-user-settings-dialog)
-- [Override default read and store functionality](#override-default-read-and-store-functionality)
+- [User Settings](#user-settings)
```

Comment:
Same comment as above, no need to add the extra categories here.

Reply:
I did not touch those lines, I assume it was added via another commit from master :/

Reply:
If you have certain VSC plugins they can automatically add changes to Markdown files when you save, maybe that could be the case?

Reply:
That might be it thanks!! :)